chat-plugins/info.js

```diff
@@ -109,6 +106,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (user.can('mute', targetUser, room) && targetUser.userid in room.users)) {
+			var bannedFrom = "";
+			for (var i = 0; i < Rooms.global.chatRooms.length; i++) {
+				var thisRoom = Rooms.global.chatRooms[i];
+				if (!thisRoom || thisRoom.isPrivate) continue;
+				if (thisRoom.bannedIps && (targetUser.latestIp in thisRoom.bannedIps || targetUser.userid in thisRoom.bannedUsers)) {
```

Comment:
Perhaps I wasn't clear enough.

Should a lobby room driver be allowed to see that a user is room banned from help, spanish, or some other room? Because I don't know if that should happen.